src/main/java/com/conveyal/analysis/controllers/AggregationAreaController.java

```diff
@@ -132,6 +132,8 @@ private FileStorageKey getStoragePath (AggregationArea area) {
 
         boolean unionRequested = Boolean.parseBoolean(query.get("union").get(0).getString());
 
+        int zoom = req.attribute("zoom") != null ? Integer.parseInt(req.attribute("zoom")) : DEFAULT_ZOOM;
```

Comment:
Just realized you're using `req.attribute` here. It should probably be sent via the request body, so here you would be able to get it via `Integer.parseInt(query.get("zoom").get(0).getString())`